src/applications/static-pages/analytics/addJumpLinkListeners.js

```diff
@@ -3,11 +3,12 @@ import { isAnchor } from './utilities';
 
 export default function addJumplinkListeners() {
   const selectors = {
-    heading: '[data-template="paragraphs/wysiwyg"] #on-this-page',
+    template: '#table-of-contents',
```

Comment:
How come you made this change? Seems like a big change! 🙂

Reply:
I wrote about in the PR description...the table of contents was typed into a WYSIWYG before but now it was changed to use the generated version of table of contents...

![image](https://user-images.githubusercontent.com/1915775/93620770-86d2b800-f9a8-11ea-8616-78830a059b6b.png)

Reply:
Oh wow... was that ever communicated? 😬

Reply:
It was just a change by a content writer. I wouldn't have expected them to communicate the change but I also don't know how I would've wrote this analytics script to capture both DOM variations

Reply:
Ahh gotcha 👍 makes sense!